src/Core/Utilities/StrictEmailAddressAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace Bit.Core.Utilities
+{
+    public class StrictEmailAddressAttribute : ValidationAttribute
+    {
+        public StrictEmailAddressAttribute()
+            : base("The {0} field is not a valid e-mail address.")
```

Comment:
Then you would need 2 different constructors, `()` (default) and another that takes a message which both would then call the underlying `EmailAddressAttribute` constructor(s) `: base()` and `: base(message)`